2023-risks/m10-insufficient-binary-protection.md

```diff
@@ -0,0 +1,102 @@
+---
+
+layout: col-sidebar
+title: "M10: Insufficient Binary Protection"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Attackers who target app binaries are motivated by various reasons. 
+
+The binary could contain valuable secrets, like commercial API keys or hardcoded cryptographic secrets that an attacker could misuse. Also, the code in the binary could be valuable on its own, e.g., because it contains critical business logic or pre-trained AI models. Some attackers might also not target the app itself but use it to explore potential weaknesses of the corresponding backend to prepare an attack.
```

Comment:
+1 for mentioning AI models.

Reply:
Another common attack pattern is application repackaging, mostly games to make personal gains. Typically payment identifiers are replaced and clones of the apps are pushed to the store or distributed using other stores.